package.json

```diff
@@ -7,15 +7,15 @@
   },
   "author": {
     "name": "Cogworks",
-    "url": "http://www.thecogworks.com/"
+    "url": "http://www.wearecogworks.com/"
```

Comment:
We're using the non-www url for that, so might be worth putting it directly with it. Minor thing, but just to highlight.